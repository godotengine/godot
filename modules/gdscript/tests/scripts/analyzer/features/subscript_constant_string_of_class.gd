class MyClass:
	static func my_static() -> void:
		print("success")

func test():
	print(MyClass["my_static"])
