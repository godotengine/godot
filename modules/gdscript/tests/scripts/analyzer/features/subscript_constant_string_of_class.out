GDTEST_OK
GDScript::my_static
